### PR TITLE
HPCC-17044 Update DZ configuration structure

### PR DESCRIPTION
**Pull request by rpastrana**
- Removes computer attribute
- Renames WhiteList to ServerList
- Hides name entry

Signed-off-by: Rodrigo Pastrana <rodrigo.pastrana@lexisnexis.com>

## Type of change:
- [ ] This change is a bug fix (non-breaking change which fixes an issue).
- [ ] This change is a new feature (non-breaking change which adds functionality).
- [x] This change is a breaking change (fix or feature that will cause existing behavior to change).
- [ ] This change alters the query API (existing queries will have to be recompiled)
-[x] This change forces configuration changes

## Checklist:
- [x ] My code follows the code style of this project.
  - [x ] My code does not create any new warnings from compiler, build system, or lint.
- [x ] My change requires a change to the documentation.
  - [ ] I have updated the documentation accordingly, or...
  - [x ] I have created a JIRA ticket to update the documentation.
  - [ ] Any new interfaces or exported functions are appropriately commented.
- [x ] I have read the CONTRIBUTORS document.
- [ ] The change has been fully tested:
  - [ ] I have added tests to cover my changes.
  - [ ] All new and existing tests passed.
  - [ ] I have checked that this change does not introduce memory leaks.
  - [ ] I have used Valgrind or similar tools to check for potential issues.
- [ ] I have given due consideration to all of the following potential concerns:
  - [x ] Scalability
  - [ x] Performance
  - [x ] Security
  - [ x] Thread-safety
  - [ ] Premature optimization
  - [ x] Existing deployed queries will not be broken
  - [ ] This change fixes the problem, not just the symptom
  - [x ] The target branch of this pull request is appropriate for such a change.
- [ ] There are no similar instances of the same problem that should be addressed
  - [ ] I have addressed them here
  - [ ] I have raised JIRA issues to address them separately

### CODE REVIEW

**Comment by rpastrana** [comment]
@xwang2713 please review. Please take a deep look at the changes to  initfiles/etc/DIR_NAME/environment.xml.in

File: initfiles/etc/DIR_NAME/environment.xml.in [hunk 1]
```diff
@@ -17,12 +17,13 @@
  <Software>
   <DropZone build="_"
             buildSet="DropZone"
-            computer="localhost"
             description="DropZone process"
             directory="${RUNTIME_PATH}/mydropzone"
             name="mydropzone"
             ECLWatchVisible="true"
             umask="022"/>
```
Comment (AttilaVamos) [inline initfiles/etc/DIR_NAME/environment.xml.in]: I think this closing tag is wrong, because it closes the <DropZone.

**Comment by jakesmith** [comment]
@rpastrana - are you still aiming this for 6.4 ? 

From what you were saying yesterday, the UI changes would take too long, so would have to pushed back ?

File: initfiles/etc/DIR_NAME/environment.xml.in (hunk 1 above)
Comment (rpastrana) [inline initfiles/etc/DIR_NAME/environment.xml.in]: Good catch.

**Comment by rpastrana** [comment]
@jakesmith this is the pull request we talked about for a long time with an ultimate decision to go forward because you felt the eclwatch/esp changes would be in place for 6.4. Are you asking to back out this change now?

**Comment by jakesmith** [comment]
@rpastrana - change looks good.

But we can't take this change without dependent Esp UI/service changes (I couldn't see an issue opened, so just opened one: HPCC-17084)
If that can't make it into 6.4, then this PR can't either.

File: initfiles/etc/DIR_NAME/environment.xml.in [hunk 2]
```diff
@@ -17,12 +17,13 @@
...
   <DropZone build="_"
             buildSet="DropZone"
-            computer="localhost"
             description="DropZone process"
             directory="${RUNTIME_PATH}/mydropzone"
             name="mydropzone"
             ECLWatchVisible="true"
-            umask="022"/>
+            umask="022">
+            <ServerList name="ServerList" server="localhost"/>
```
Comment (AttilaVamos) [inline initfiles/etc/DIR_NAME/environment.xml.in]: Is "localhost" really can be used as a server address? How this can be resolve in multinode system?

Comment (jakesmith) [inline initfiles/etc/DIR_NAME/environment.xml.in]: It wouldn't make a lot of sense in multinode systems, but it does on 1 node system.
However, we tend to use ".", meaning 1st preferred NIC ip, which would avoid this issue.

Comment (rpastrana) [inline initfiles/etc/DIR_NAME/environment.xml.in]: This is the default config for one-node cluster.
@jakesmith good point.

**Comment by rpastrana** [comment]
@jakesmith please review.

**Comment by jakesmith** [comment]
@rpastrana - looks good.

@richardkchapman - ready to merge.

**Comment by richardkchapman** [comment]
Waiting to see if HPCC-17084 ends up mergable...

**Comment by rpastrana** [comment]
@richardkchapman this one can be merged now. As I understand, Attila's recent PR relies on this change.

**Pull request merged**